plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -1279,6 +1285,27 @@ private static DiskTO getIsoDiskTO(DiskTO[] disks) {
         return null;
     }
 
+    private CopyFileInVmAnswer execute(CopyFileInVmCommand cmd) {
+        VmwareManager mgr = getServiceContext().getStockObject(VmwareManager.CONTEXT_STOCK_NAME);
+        File keyFile = mgr.getSystemVMKeyFile();
+        try {
+            File file = new File(cmd.getSrc());
```

Comment:
Does file object consumes a resource to be flushed or closed post the usage or its auto closed?